plugins/kapt3/kapt3-compiler/test/org/jetbrains/kotlin/kapt3/test/AbstractKotlinKapt3IntegrationTest.kt

```diff
@@ -123,6 +131,24 @@ abstract class AbstractKotlinKapt3IntegrationTest : KotlinKapt3TestBase() {
         }
     }
 
+    override fun loadMultiFiles(files: List<TestFile>) {
+        val project = myEnvironment.project
+        val psiManager = PsiManager.getInstance(project)
+
+        val tmpDir = KtTestUtil.tmpDir("kaptTest")
```

Comment:
Delete this dir in tearDown()?

Reply:
I copied this from AbstractKotlinKapt3Test which does not do anything in teardown. Should I update both? (not sure what is the preferred way in these tests)
https://github.com/JetBrains/kotlin/blob/master/plugins/kapt3/kapt3-compiler/test/org/jetbrains/kotlin/kapt3/test/AbstractKotlinKapt3Test.kt#L77

Reply:
I looked at how temp dir is used in other tests and it doesn't seem that it get deleted often. Ideally you should delete it ofc, but I wouldn't fix it everywhere right now.